ansible/roles/iroha-docker/defaults/main.yml

```diff
@@ -18,27 +18,24 @@ iroha_forced_cleaning: False
 # Default: false
 iroha_custom_hostnames: false
 
-# Affects how Iroha peers are communicated. If set to `true`, Docker overlay network with that
-# name will be used. It must be created beforehand. The recommended way is to use Swarm cluster
-# or create a network manually using a KV storage.
-# Setting it to `true` without creating the network will only work for a single host deployment
-# (Iroha peers will only be able to communicate within that host). Suitable for local-only
-# deployments for testing purposes.
+# Affects how Iroha peers are communicated.
+# If set to `internal` Iroha will use docker bridge network to communicate between nodes. You can only use this mode if you want to deploy all nodes on one host  
+# If set to `overlay` Iroha will create a docker overlay network to communicate between node.
+# If set to `external` Iroha will map the port to host and will use it to communicate. Overlay network plugin (like Calico) or docker swarm should be enabled on all host.
 #
-# Default: false
-iroha_overlay_network: false
+# Default: external
+iroha_network: external
```

Comment:
I am not sure that `external` default is a good idea because swarm or Calico may not be installed. Maybe use `overlay` by default?
Or there is a discrepancy in documentation and `overlay` and `external` should be swapped, so that `overlay` is for swarm or Calico, and `external` is for docker overlay network.

Reply:
this is discrepancy in documentation